src/error/generated.ts

```diff
@@ -406,11 +406,9 @@ export namespace AuthorizationErrors {
    */
   export class RateExceededError extends TwilioError {
     /**
-     * Message payload size limit exceeded.
```

Comment:
Will this be added back in once 31209 is fixed?

Reply:
Yes, see ticket VBLOCKS-3144. I thought it best to just avoid generating it for now. It will get fixed soon.